jbpm-flow-builder/src/main/java/org/jbpm/compiler/xml/XmlProcessReader.java

```diff
@@ -101,4 +117,19 @@ void setProcesses(final List<Process> processes) {
     public ProcessBuildData getProcessBuildData() {
         return (ProcessBuildData) this.parser.getData();
     }
+
+    protected String processParserMessage(Object parent, Attributes attr, String errorMessage) {
+        String nodeId = (attr == null  || attr.getValue("id") == null) ? "" : attr.getValue("id");
+        String nodeName = (attr == null  || attr.getValue("name") == null) ? "" : attr.getValue("name");
+
+        if(parent != null && parent instanceof RuleFlowProcess) {
```

Comment:
@tiho I guess this would only work for nodes that are inside the top-level process, not for composite nodes?